index.html

```diff
@@ -1210,6 +1300,16 @@ <h3>
             Updating the manifest
           </h3>
           <aside class="issue" data-number="446"></aside>
+          <p>
+            User agents MAY, at any time, re-fetch and update a manifest for an
```

Comment:
These are great insights, and we are either currently doing this, or plan on doing this, in Chromium.

1. I have a doc describing when exactly Chromium applies the manifest update here [1]. Basically - we wait until there are no app windows open. (edge case - if you load the page and there are no app windows open, then we update immediately). The only caveat is that we have a 1 day (in-memory) throttle, so a manifest cannot be considered for update more than once a day.
2. We currently (generally) do not allow name and icon updates, and we have a project that is currently being worked on the prompt the user for these updates. The user can reject name & icon updates. However, we do automatically update other members of the manifest, like theme color, etc. It might be good to provide language here that a user agent is not required to apply any of the updated manifest fields, which given user agents flexibility for privacy & security concerns.

[1] https://docs.google.com/document/u/1/d/1q5kmxNU7i4eem22LouMaIJ6123jOw5_zuxVQW1wfW8Y/edit

Reply:
Ok, that's great to hear @dmurph. For the spec, I think we should have a working assumption that the manifest is updated on every page load, and then make a provision to support custom update behavior that user agents might use. 

We should document the concerns around icons and name in the privacy and security sections.

Reply:
Updated the wording, can you take another look?